kie-api/src/build/revapi-config.json

```diff
@@ -1,5 +1,16 @@
 {
   "revapi" : {
+    "java" : {
+      "filter" : {
+        "classes" : {
+          "regex" : true,
+          /* Not used. The class is not referenced in any of the downstream (Drools, jBPM) code. The Activator itself was disabled in 2013.
+             Exclusion is here in order not to check the transitive classes from dependencies which this class used. This exclude can be removed once
+             this class is not present in the "oldVersion" of artifact, i.e. 7.0.0.Final */
```

Comment:
I am afraid JSON does not support comment like these. Revapi seems to cope with that, but I guess we should fix this to make the file compliant to the JSON standard.